README.md

```
@@ -12,4 +12,19 @@ set -a
 source .env
 set +a
 ./gradlew bootRun
-```
\ No newline at end of file
+```
+
+### kafka, kafka-ui 실행 방법
+
+- kafka-ui 는 배포 환경에서는 필요하지않을 수 있어, profile `ui` 활성화 시에만 실행되도록 설정했습니다.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)